input/fsh/omrs-patient.fsh

```diff
@@ -33,3 +33,43 @@ Id:           omrs-patient-identifier-location
 Title:        "OpenMRS Patient Identifier Location"
 Description:  "OpenMRS location for which this identifier is valid"
 * value[x] only Reference(OMRSLocation)
+
+// OpenMRS Patient  Example
+
+Instance: example-openmrs-Patient
```

Comment:
This definition is causing the following error in the CI: 

```
Sushi: error Element Patient.identifier.extension has minimum cardinality 1 but occurs 0 time(s). (00:05.354)
Sushi: File: /github/workspace/input/fsh/omrs-patient.fsh                        (00:05.355)
Sushi: Line: 39 - 58                                                             (00:05.358)
Sushi: error Element Patient.identifier.extension:location has minimum cardinality 1 but occurs 0 time(s). (00:05.359)
Sushi: File: /github/workspace/input/fsh/omrs-patient.fsh                        (00:05.361)
```

@sherrif10 you might need to add a location extension to the identifier definition.

Reply:
Sure thank you, i updated it with location extension.